tango_simlib/tango_sim_generator.py

```diff
@@ -297,8 +298,18 @@ def _reset_to_default_state(self):
                                 else:
                                     adjustable_val = float(sim_quantity_meta_info[attr])
 
-                        simulated_quantity.attr = adjustable_val
-                
+                        if attr == 'last_val':
+                            simulated_quantity.last_val = adjustable_val
+                        elif attr == 'mean':
+                            simulated_quantity.mean = adjustable_val
+                        elif attr == 'std_dev':
+                            simulated_quantity.std_dev = adjustable_val
+                        elif attr == 'max_slew_rate':
+                            simulated_quantity.max_slew_rate = adjustable_val
+                        elif attr == 'min_bound':
+                            simulated_quantity.min_bound = adjustable_val
+                        elif attr == 'max_bound':
+                            simulated_quantity.max_bound = adjustable_val
 
```

Comment:
Should we raise an exception if it is none of these options?  Also, code could be a little shorter if we do a `hasattr` check and then a  `setattr` to work more generically.

Reply:
We do not need to raise an exception. We get them from https://github.com/ska-sa/tango-simlib/blob/fa6532247d2f4f8e633c7f863fd5cef05eb9d623/tango_simlib/tango_sim_generator.py#L248-L249

and loop through the list.